src/variables/mask.js

```diff
@@ -0,0 +1,9 @@
+// Brazilian CPF's Regex
+export const cpfMask = value => {
+  return value
+    .replace(/\D/g, "") // replaces any character that is not number for nothing
+    .replace(/(\d{3})(\d)/, "$1.$2") // captures 2 groups of numbers the first of 3 and the second of 1, after capturing the first group it adds a point before the second group of numbers
```

Comment:
Did you created this pattern ?

Reply:
Nope. Source [here.](https://medium.com/trainingcenter/mascara-de-cpf-com-react-javascript-a07719345c93)